automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -546,6 +538,8 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
 
       withBillingProject("auth-domain-spec") { projectName =>
         api.billing.addUserToBillingProject(projectName, user.email, BillingProjectRole.Owner)
+        register cleanUp api.billing.removeUserFromBillingProject(projectName, user.email, BillingProjectRole.Owner)
```

Comment:
The clean-up should be done most of the time. I think it would be helpful to add a function to `BillingFixtures` to take care of this so we don't have to worry as much about remembering to do it. Something like:
```
trait BillingFixtures ... {
  ...
  def addUserToBillingProject(projectName: String, email: String, role: BillingProjectRole)(implicit token: AuthToken): Unit = {
    Orchestration.billing.addUserToBillingProject(projectName, email, role)
    register cleanUp Orchestration.billing.removeUserFromBillingProject(projectName, email, role)
  }
}
```

Reply:
`withBillingProject` refactored to optionally allow add/remove user to a Billing project.
https://github.com/broadinstitute/workbench-libs/pull/82